src/Sarif.Viewer.VisualStudio/OpenLogFileCommands.cs

```diff
@@ -168,117 +165,84 @@ private async System.Threading.Tasks.Task MenuItemCallbackAsync(object sender, E
 
             if (logFile == null)
             {
-                string title = "Open Static Analysis Results Interchange Format (SARIF) file";
-                string filter = "SARIF files (*.sarif)|*.sarif";
+                FieldInfo[] toolFormatFieldInfos = typeof(ToolFormat).GetFields();
+                List<KeyValuePair<FieldInfo, string>> fieldInfoToOpenFileDialogFilterDisplayString = new List<KeyValuePair<FieldInfo, string>>(toolFormatFieldInfos.Length);
```

Comment:
>List<KeyValuePair<FieldInfo, string>> [](start = 16, length = 37)

Per this code bases's conventions, this can be `var` since the type is explicit with `new` on the RHS.